templates/rtinfo/rtinfo.py

```diff
@@ -0,0 +1,52 @@
+from jumpscale import j
+from zerorobot.template.base import TemplateBase
+from zerorobot.template.state import StateCheckError
+
+NODE_CLIENT = "local"
+
+
+class Rtinfo(TemplateBase):
+    version = "0.0.1"
+    template_name = "rtinfo"
+
+    def __init__(self, name, guid=None, data=None):
+        super().__init__(name=name, guid=guid, data=data)
+        self.add_delete_callback(self.uninstall)
+
+    def validate(self):
+        # validate that rtinfo client is running when installed
+        self._monitor()
+
+    def _monitor(self):
```

Comment:
Usually the monitor method is a recurring action.
Here again you can make it recurring and check every X minutes that the rtinfo process is still present, if not, restart it.